lib/Mitigation.pm

```diff
@@ -246,6 +247,7 @@ sub lscpu {
 #This function will finish testing in default status.
 #As out of box testing. and clean up all mitigations parameters.
 sub check_default_status {
+    diag "check mitigation default status.";
```

Comment:
Since we're adding logs, think we could have the format uniform, like all logs first letter uppercase